modules/ROOT/pages/view.adoc

```diff
@@ -29,6 +32,8 @@ Only users with the *Visualizer Editor* permission can create defined views.
 . Click *+New*.
 . In the *New view* dialog, enter a *Name* for the view, and click *Save*.
 . Configure the view by selecting which environments, xref:use-tags-in-visualizer.adoc[tags], and services to include.
+. Hide the nodes which you have selected in the canvas using the *Hide selected services* option.
```

Comment:
This is just one, simple way to configure the view (step #4). They are not necessary steps in defining a view. Is there a way to communicate this/show them at sub-bullets?

Reply:
I will ask about sub bullets. I am not actually sure.